packages/standalone/public/viewer/js/modules/graph.js

```
@@ -0,0 +1,1025 @@
+    // Filtering
+    // =============================================
+    /**
+     * Populate topic filter dropdown
+     */
+    populateTopicFilter(topics) {
+        const datalist = getElementByIdOrNull('topic-filter-list');
+        if (!datalist) {
+            // Fallback for old select element
+            const select = getElementByIdOrNull('topic-filter');
+            if (!select)
+                return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)